src/pages/Writer/Writer.vue

```diff
@@ -1,76 +1,218 @@
+      >
+        <button
+          class="formatting-option"
+          :class="{'is-active': editor.isActive('bold')}"
+          @click="editor.chain().focus().toggleBold().run()"
+        >
+          Bold
+        </button>
+        <button
+          :class="{'is-active': editor.isActive('italic')}"
+          @click="editor.chain().focus().toggleItalic().run()"
+        >
+          Italic
+        </button>
+        <button
+          :class="{'is-active': editor.isActive('strike')}"
+          @click="editor.chain().focus().toggleStrike().run()"
+        >
+          Strike
+        </button>
+      </bubble-menu>
+    </div>
+    <editor-content :editor="editor" class="writer__editor" />
+  </div>
+  <button class="writer__post-button" @click="onPost">
+    Post
+  </button>
   </div>
 </template>
 
 <script>
+import {Editor, EditorContent, BubbleMenu} from '@tiptap/vue-3'
+import StarterKit from '@tiptap/starter-kit'
+import Document from '@tiptap/extension-document'
+import Placeholder from '@tiptap/extension-placeholder'
+import { Notify } from 'quasar'
 import './Writer.scss'
 
+const CustomDocument = Document.extend({
+  content: 'heading block*'
+})
+
 export default {
   name: 'Writer',
+  components: {
+    EditorContent,
+    BubbleMenu
+  },
+
   data() {
     return {
-      title_input: '',
-      tag_input: '',
-      content_input: '',
+      editor: null
     }
   },
+
+  mounted() {
+    this.editor = new Editor({
+      extensions: [
+        CustomDocument,
+        StarterKit.configure({
+          document: false
+        }),
+        Placeholder.configure({
+          placeholder: ({ node }) => {
+            if (node.type.name === 'heading') {
+              return 'What’s the title?'
+            }
+            return 'Can you add some further context?'
+          },
+        }),
+      ],
+      content: `
+        <h1 style="font-weight: bold">
+          Add a heading
+        </h1>
+        <p>
+          Start writing here
+        </p>
+      `
+    })
+  },
+
+  beforeUnmount() {
+    this.editor.destroy()
+  },
   methods: {
-    async sendPost() {
-      const userBlog = {'Title': this.title_input, 'Tags': this.tag_input, 'Content': this.content_input, 'Date': new Date()}
-      console.log(userBlog)
-      if (!this.title_input) {
-        return
+    async onPost() {
+      const blogContent = this.editor.getJSON()
+      const blogTitle = blogContent.content[0].content[0]['text']
+      const blogBody = blogContent.content[1].content[0]['text']
+      if (blogTitle === 'Add a heading')
+        Notify.create({
+          message: `Add a title to your post first!`,
+          color: 'negative'
+        })
+      else {
+        if (blogBody === 'Start writing here')
+          Notify.create({
+            message: `Write your blog before posting!!`,
+            color: 'negative'
+          })
+        else
+          await this.$store.dispatch('sendPost', { message: JSON.stringify(blogContent) })
```

Comment:
The user should be notified on successful posting of the blog and even redirected to `My Blogs` section. Will create an issue on the same